java/CustomMetrics/RecordCount/src/main/java/com/amazonaws/services/msf/RecordCountApplication.java

```diff
@@ -0,0 +1,101 @@
+    package com.amazonaws.services.msf;
+     
+    import com.amazonaws.services.kinesisanalytics.runtime.KinesisAnalyticsRuntime;
+    import java.util.Map;
+    import java.util.Properties;
+    import org.apache.flink.api.common.functions.FilterFunction;
+    import org.apache.flink.api.common.functions.RichMapFunction;
+    import org.apache.flink.api.common.serialization.SimpleStringSchema;
+    import org.apache.flink.configuration.Configuration;
+    import org.apache.flink.connector.kinesis.sink.KinesisStreamsSink;
+    import org.apache.flink.metrics.Gauge;
+    import org.apache.flink.streaming.api.datastream.DataStream;
+    import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+    import org.apache.flink.streaming.connectors.kinesis.FlinkKinesisConsumer;
+    import org.slf4j.Logger;
+    import org.slf4j.LoggerFactory;
+     
+    /**
+     * A sample Managed Service For Apache Flink application with Kinesis data streams as source and sink with simple filter
+     * function.
+     */
+    public class RecordCountApplication {
+     
+        private static final Logger LOG = LoggerFactory.getLogger(RecordCountApplication.class);
+     
+        public static void main(String[] args) throws Exception {
+            StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+            Map<String, Properties> runConfigurations = KinesisAnalyticsRuntime.getApplicationProperties();
```

Comment:
This should be changed following the pattern of the other examples.
We should stop having examples that only runs on MSF, unless there is a very compelling reason.

Reply:
added local execution